README.md

```diff
@@ -37,11 +36,42 @@ communications. Therefore this repository serves primarily to document
 how I believe people should act in the Stack community.
 
 As I have done in the past, I will be available for people to
-privately express concerns about the behaviors of others, in keeping
-with the CoC. And while I will not make unilateral decisions on behalf
-of the Stack leadership, I will, as necessary:
+privately express concerns about the behaviors of others. And while I
+will not make unilateral decisions on behalf of the Stack leadership,
+I will, as necessary:
 
 * Reach out privately to request changes in behavior
 * Publicly request changes in behavior
 * Raise the issue with the rest of the Stack leadership and consider a
   more official response
+
+## Code of Conduct
+
+Following the discussion on [issue
+#1](https://github.com/snoyberg/stack-coc/issues/1), it does not seem
+to be appropriate to use a standard Code of Conduct for this
+repository. The reason is that this repository is an unofficial set of
+guidelines, reflecting one person's recommendations and offer to
+intervene in cases where concerning behavior occurs.
```

Comment:
I understand that you don't want to overstep your authority and that you cannot speak for others. But would it be possible to reword this in a way which would make it easy for others to join in? I am excited about this repository because I believe it has the potential to spark a movement within the Haskell community. My hope is that a large fraction of the community will state that they agree with these guidelines, and that they will become our community's de facto norm. The `PARTICIPANTS.md` file already hints towards such a future. So how about:

> The reason is that the guidelines listed in this repository are not imposed by an authority onto others, but rather embraced by the members of the community listed in `PARTICIPANTS.md`. If you use Haskell (even if you don't use stack!) and you agree with these guidelines, please send a PR to add your name to the list, to help us signal that the whole community supports these guidelines.

Reply:
While we're on the topic of adding names to `PARTICIPANTS.md`: I think more people will participate if it is easier to do so, e.g. by adding a :+1: on an existing PR rather than by submitting a new PR.

Reply:
Interesting idea. I'm not sure about the wording quite yet though:

* So far everything has been "Stack community." Do we want to say "Haskell" here? Would it make more sense to just let _anyone_ add their name, regardless of affiliation in any community?
* "signal that the whole community supports these guidelines" would be overstepping. I don't think we're trying to claim that the whole community, or even a majority, or even a significant minority supports this. Perhaps something like "signal that you want to help with improving communication"?
* I understand that a 👍 is easier to add, but I'm concerned it might make it too easy, and not have much bearing. It also means that there's no real record of people withdrawing their support (if they disagree with changes made) or when they started signing up.

Reply:
> So far everything has been "Stack community." Do we want to say "Haskell" here?

Well, I for one am a proud member of the Haskell community, but I don't feel nearly as much attachment to the Stack brand, even though I use `stack` every day. Furthermore, since the needlessly heated rivalry between some proponents of stack and cabal is a big reason why I think the Haskell community needs to collectively adopt communication guidelines, the future I am excited about is definitely one in which prominent members from both of those "sides" would sign off on the same communications guidelines.

I now realize that I have naively assumed that your goals were there same as mine, and that I have imagined all sorts of contrivances in order to explain why you keep mentioning Stack in these documents. I had imagined that in the past you had received some criticism for trying to speak for the whole community, and that this time you wanted to pre-empt similar criticism by pretending to only speak for the Stack community; and then restricting the set of people on behalf of whom you speak even further by saying you only speak for yourself. I guess I must now re-read your blog post and attempt to interpret it according to what it says, not what I want it to say.

> Would it make more sense to just let anyone add their name, regardless of affiliation in any community?

No, I think that would be so broad as to make it meaningless. I don't think having a long list of random humans sign a document is going to encourage non-signees to change their behavior, because they don't feel related to those who did sign. But if all the big names of a community sign a document, I think that has the potential to impact those who consider themselves a member of that particular community.

> "signal that the whole community supports these guidelines" would be overstepping.

Hmm, you're right, I again confused my hopes with reality there. My hope is that the whole community will join this movement and that this will in turn influence the behavior of those who want to be a part of that community. The reality is that this is still a proposal and that your name is currently the only one on the list, and so it is definitely not appropriate to claim otherwise. I still think that the goal of this part of the sentence should be to emphasize that it is only by having lots of signatures that the proposal can have an impact. How about "please send a PR to add your name to the list, as a longer list legitimizes these guidelines and makes it more likely for a violator to accept that they should change their behavior"?

Reply:
Let's hold off for just the moment on the participants piece. I'd rather get the basics in place, and then we can discuss whether or not it makes sense to encourage others to join, and how to word things around that.

I won't beat around the bush: in my ideal world, the entire Haskell world would agree to some kind of guidelines like I'm proposing (though maybe not exactly what I've said, I recognize that I'm somewhat opinionated here). But I'm neither asking anyone to do so, nor trying to maneuver   this repository into such a position in the future.

Reply:
All right, if you don't want to encourage others to join yet, then I don't think this paragraph needs any changes.